docs/pages/versions/unversioned/guides/customizing-webpack.md

```diff
@@ -2,28 +2,55 @@
 
-You will more than likely need to customize your webpack config in a large scale or production application. The best way to do this is to extend or rebuild the webpack config using methods exported from `@expo/webpack-config`.
+> This is akin to `react-scripts` & `create-react-app`.
 
-By default `expo-cli` checks to see if a `webpack.config.js` files exists in the root directory before using `@expo/webpack-config`. This should provide you with a simple interface for creating complex configurations.
+If you need to edit the config the best way to do this is by running `expo customize:web` and selecting the `webpack.config.js` option.
+This will install `@expo/webpack-config` as a devDependency and create a template `webpack.config.js` into your project.
+You can now make changes to the default config then return it for the CLI to use.
+Deleting the config will cause Expo to fallback on the default again.
 
-## Extending the Default Config
+If you create a new Webpack config or make any changes to it you'll need to restart your Webpack dev server with `expo start --web`.
 
-Say you want to add a file type that isn't supported by default.
-Simply create a `webpack.config.js` in your project root and merge in a new loader.
+## Example
 
 **`webpack.config.js`**
 
 ```ts
-const merge = require('webpack-merge');
-// This will automatically get the dev/prod config based on process.env.NODE_ENV.
-const expoConfig = require('@expo/webpack-config');
-
-// Expo expects a function so we can pass around options.
-module.exports = function(env, argv) {
-  return merge(expoConfig(env, argv), {
-    // It's clobbering time...
-  });
+const createExpoWebpackConfigAsync = require('@expo/webpack-config');
+
+// Expo CLI will await this method so let's future proof
+// it by using await on the default config.
+module.exports = async function(env, argv) {
+  const config = await createExpoWebpackConfigAsync(env, argv);
+  // If you want to add a new alias to the config.
+  config.resolve.alias['moduleA'] = 'moduleB';
+
+  // Maybe you want to turn off compression in dev mode.
+  if (config.mode === 'development') {
+    config.devServer.compress = false;
+  }
+
+  // Or prevent minimizing the bundle when you build.
+  if (config.mode === 'production') {
+    config.optimization.minimize = false;
+  }
+
+  // Finally return the new config for the CLI to use.
+  return config;
 };
 ```
+
+# Editing static files
+
+You can use `expo customize:web` to create a replacement `index.html`, `serve.json`, `favicon.ico`, etc...
+If you want to reset these files to their initial values simply run `expo customize:web --force` or `-f` for short. All of the files you select will be copied to a `web/` folder in your project's root directory. Think of this folder like `public/` in Create React App. We use "web" instead of "public" because Expo projects target more then just web. In React Native we similarly put non-agnostic project files in `/ios` and `/android` folders.
```

Comment:
```suggestion
If you want to reset these files to their initial values simply run `expo customize:web --force` or `-f` for short. All of the files you select will be copied to a `web/` folder in your project's root directory. Think of this folder like `public/` in Create React App. We use "web" instead of "public" because Expo projects target more then just web. For mobile platforms, we similarly put platform-specific project files in `/ios` and `/android` folders.
```

Reply:
this section is confusing. you talk about resetting files, and then about copying files to the web folder. these seem like independent concepts.